plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -488,10 +489,6 @@ global proc mayaUsdTranslatorExport_SetFromOptions(string $currentOptions, int $
                 mayaUsdTranslatorExport_SetCheckbox($optionBreakDown[1], $enable, "exportDisplayColorCheckBox");
             } else if ($optionBreakDown[0] == "exportInstances") {
                 mayaUsdTranslatorExport_SetOptionMenuByBool($optionBreakDown[1], $enable, "exportInstancesPopup");
-            } else if($optionBreakDown[0] == "exportCameras"){
-                mayaUsdTranslatorExport_SetOptionMenuByBool($optionBreakDown[1], $enable, "exportInstancesPopup");
-            } else if($optionBreakDown[0] == "exportLights") {
-                mayaUsdTranslatorExport_SetOptionMenuByBool($optionBreakDown[1], $enable, "exportInstancesPopup");

```

Comment:
Why were these lines removed?

Reply:
Those lines are some debug code I used before and forgot to remove. They are totally useless now...

Reply:
Why would they be useless. Someone can call this function adding those strings to the input options to control the starting value of these two export options.

Reply:
Oh no those flags are not longer available. Now we use `excludeExportTypes` flag, so it will be `excludeExportTypes=[Cameras, Lights]` like this. These two flags are abandoned. It's my fault I forgot to delete them here...